### ISSUE TITLE
Can we have the source of the server please ?

### ISSUE THREAD

**Issue by BararDiHoumes** [miroslavpejic85/p2p#35]
Sorry for asking this way, but can we have the source of the server , so we can create our own nodes ?
Thank You in advance dear Miroslav.

**Comment by IanMcLumber** [comment]
Yes please. Using external server for something quite this dangerous is really making me scary. 
Please provide the server source.
